src/components/molecules/Common/Header/index.tsx

```diff
@@ -105,15 +105,19 @@ const Header: React.FC<Props> = ({
 
 const Wrapper = styled.header`
   width: 100%;
-  height: ${metrics.headerHeight}px;
+  min-height: ${metrics.headerHeight}px;
```

Comment:
With this changed to min-height, does the header  grow its height at any point as the width decreases because of children bunching up?

Reply:
yes, sometimes when the project name is a long name. when screen size became smaller, the project name will jump out of the header bar, witch is quite terrible. So we did nothong to handle the project name length for now, I think it's better to not limite the header height, so the project name will not jump out of header